spec/controllers/activities_controller_spec.rb

```diff
@@ -59,7 +59,7 @@
                                                    project_id: work_package.project_id))
       }
 
-      before { get 'index' }
+      before do get 'index' end
```

Comment:
Prefer {...} over do...end for single-line blocks.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.